engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/desugar/OperatorToFunction.scala

```diff
@@ -44,6 +44,11 @@ case object OperatorToFunction extends IRPass {
     ir: Module,
     moduleContext: ModuleContext
   ): Module = {
+    if (!java.lang.Boolean.getBoolean("testing.mini.passes")) {
```

Comment:
We don't want to use this code in production, but we need it in test. This trick with a boolean property is ugly and we should get rid of it - probably by moving the code under this check to a test code. CCing @Akirathan

Reply:
The original mega pass implementation moved to test sources in 1aaa58925f19a70aaf63ba29d27618f5a0bd6790